client/src/locales/translations/en_GB.json

```diff
@@ -832,7 +832,8 @@
     "tryAgainBtn": "Try Again"
   },
   "unsubscribe": {
-    "notifyPrefs": "Notification Preference",
```

Comment:
@ddveloper Just FYI, we shouldn't be updating any other translation file other than `en_US.json` since Lokalise will automatically create a pull request to translate from `en_US.json` to the other languages. See https://github.com/FightPandemics/FightPandemics#internationalization

Reply:
Got it, thanks Manny, I will create PR next time 👍